x-pack/plugins/synthetics/server/routes/synthetics_service/run_once_monitor.ts

```diff
@@ -37,17 +36,15 @@ export const runOnceSyntheticsMonitorRoute: SyntheticsRestApiRouteFactory = () =
 
     const paramsBySpace = await syntheticsService.getSyntheticsParams({ spaceId });
 
-    const errors = await syntheticsService.runOnceConfigs([
-      formatHeartbeatRequest({
-        // making it enabled, even if it's disabled in the UI
-        monitor: { ...validationResult.decodedMonitor, enabled: true },
-        monitorId,
-        heartbeatId: monitorId,
-        runOnce: true,
-        testRunId: monitorId,
-        params: paramsBySpace[spaceId],
-      }),
-    ]);
+    const errors = await syntheticsService.runOnceConfigs({
+      // making it enabled, even if it's disabled in the UI
```

Comment:
Why remove formatHeartbeatConfigs?

Reply:
it has been moved inside synthetics service class to unify.